release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -1036,6 +1041,52 @@ Support for running {op-system-base-full} 7 compute machines in {product-title}
 
 You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. You must deploy new {op-system-base} 8 hosts, and the old {op-system-base} 7 hosts must be removed.
 
+[id="ocp-4-10-removed-feature-azure-mint-mode"]
+==== Support for minting credentials for Microsoft Azure removed
+
+Support for using the Cloud Credential Operator (CCO) in mint mode on Microsoft Azure clusters has been removed. This change is due to the planned link:https://azure.microsoft.com/en-us/updates/update-your-apps-to-use-microsoft-graph-before-30-june-2022/[retirement of the Azure AD Graph API by Microsoft on 30 June 2022], and is being backported to all supported versions of {product-title} in z-stream updates.
+
+For previously installed Azure clusters that use mint mode, the CCO attempts to update existing secrets. If a secret contains the credentials of previously minted app registration service principals, it is updated with the contents of the secret in `kube-system/azure-credentials`. This behavior is similar to passthrough mode.
+
+[NOTE]
+====
+In addition to the `Contributor` role that is required by mint mode, the app registration service principals require the `User Access Administrator` role that is used for passthrough mode.
+====
+
+While the Azure AD Graph API is still available, the CCO attempts to clean up previously minted app registration service principals.
```

Comment:
So, does this mean that users could/should do something that is not the next few steps to avoid their app registration service principals getting orphaned in July?